packages/builder/src/index.js

```diff
@@ -1 +1,10 @@
+import Builder from './builder'
```

Comment:
Can be `export default Builder` ?

Reply:
cc @pi0

Reply:
The way of exporting classes can be improved in another PR :)